api/routes/auth.js

```diff
@@ -1,11 +1,14 @@
-import Express from "express";
-import { logout, signin, signup, verifyEmail } from "../controller";
-const AuthRouter = new Express.Router();
+const Express = require("express");
+const AuthRouter = Express.Router();
+const {logout, signin, signup, verifyEmail, forgotPassword,resetPassword }=require("../controller/auth");
```

Comment:
please don't change anything from export-import to require as I am using Babel to handle this.